weblate/trans/templatetags/translations.py

```diff
@@ -65,7 +65,13 @@
 
 GLOSSARY_TEMPLATE = """<span class="glossary-term" title="{}">"""
 
-WHITESPACE_RE = re.compile(r"(  +| $|^ )", re.MULTILINE)
+
+WHITESPACE_REGEX = (
+    r"(  +| $|^ |\u00A0|\u1680|\u2000|\u2001|\u2002|\u2003|"
+    + r"\u2004|\u2005|\u2006|\u2007|\u2008|\u2009|\u200A|"
+    + r"\u202F|\u205F|\u3000)"
```

Comment:
Have you tested this? With the `r` prefix, it will IMHO look for `u3000` literal instead of the Unicode chars.